internal/engine/upper.go

```diff
@@ -107,11 +107,6 @@ func (u Upper) Start(
 	}
 
 	var manifestNames []model.ManifestName
```

Comment:
is this never populated? do we use it anywhere?

Reply:
oooops